cftime/_cftime.pyx

```diff
@@ -332,20 +318,42 @@ DATE_TYPES = {
 }
 
 
-def to_calendar_specific_datetime(datetime, calendar, use_python_datetime):
+#def to_calendar_specific_datetime(dt, calendar, use_python_datetime):
+#    if use_python_datetime:
+#        return real_datetime(
+#               dt.year,
+#               dt.month,
+#               dt.day,
+#               dt.hour,
+#               dt.minute,
+#               dt.second,
+#               dt.microsecond)
+#    else:
+#        return datetime(
+#               dt.year,
+#               dt.month,
+#               dt.day,
+#               dt.hour,
+#               dt.minute,
+#               dt.second,
+#               dt.microsecond,
+#               calendar=calendar)
```

Comment:
The commented out version does not return the calendar-specific subclasses.  Since we might want to remove those sub-classes in a future release, I left it in there.

Reply:
Gotcha -- it makes sense to leave it in then.